modules/restart-scanner-and-scannerdb-containers.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * configuration/reissue-internal-certificates.adoc
+
+:_module-type: PROCEDURE
+[id="restart-scanner_{context}"]
+= Restarting the Scanner and ScannerDB containers
+
+[role="_abstract"]
+You can restart the Scanner and ScannerDB container by deleting the pods.
+
+.Procedure
+
+* Run the following command to delete the Scanner and ScannerDB pods:
```

Comment:
```suggestion
* Delete the Scanner and ScannerDB pods, run the following command:
```

See the [code blocks command syntax](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output). It should be similar to: 

> Step description, run the following command:

Reply:
@sagidlow @gaurav-nelson I do not agree that either can be used. "Delete the Scanner and ScannerDB pods, run the following command" is a comma splice.